contracts/base/Loans.sol

```diff
@@ -361,4 +361,15 @@ contract Loans is LoansInterface, SignerRole {
             liquidated: false
         });
     }
+
+    function isCollateralSentEnough(
```

Comment:
This function isnt used anywhere? Was it meant to be incorporated to the actual contract?

Reply:
This function can be used in the takeOutLoan function (where the borrower sends the collateral).

As I changed the configuration for the Chainlink Oracle, I tested it before using it (in takeout loan).